scipy/stats/_stats_py.py

```diff
@@ -7920,10 +7956,20 @@ def ks_2samp(data1, data2, alternative='two-sided', method='auto'):
 
     Returns
     -------
-    statistic : float
-        KS statistic.
-    pvalue : float
-        One-tailed or two-tailed p-value.
+    res: KstestResult
+        An object containing attributes:
+
+        statistic : float
+            KS test statistic.
+        pvalue : float
+            One-tailed or two-tailed p-value.
+        statistic_location: float
+            Value of the stochastic variable closest to the KS statistic. By
+            definition, the distance between the two sampled distributions at
+            this value is largest.
+        statistic_sign: int
+            +1 if the first distribution is above the second at the KS
+            statistic location, otherwise -1.
```

Comment:
```suggestion
            +1 if the empirical distribution function of `data1` exceeds
            the empircal distribution function of `data2` at 
            `statistic_location`, otherwise -1.
```